target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -498,6 +498,17 @@ define Device/tplink_tl-mr6400-v1
 endef
 TARGET_DEVICES += tplink_tl-mr6400-v1
 
+define Device/tplink_tl-wa1201-v2
+  $(Device/tplink-safeloader-uimage)
+  SOC := qca9563
+  IMAGE_SIZE := 15872k
```

Comment:
That's not 0xfa0000

Reply:
Indeed. I computed this incorrectly. I now correctly imported the tplink firmware layout for the tplink specific partitions. The first tplink partition starts at `0xd10000`, the firmware partition starts at `0x30000`, so `IMAGE_SIZE` will be `0xce0000` which is `13184k`